src/commcare_cloud/terraform/staging/terraform.tf

```diff
@@ -27,57 +26,57 @@ module "commcarehq" {
     {
       server_name           = "web0-staging"
       server_instance_type  = "t2.micro"
-      subnet_index          = 0
+      subnet                = "private-a"
     },
     {
       server_name           = "web1-staging"
       server_instance_type  = "t2.micro"
-      subnet_index          = 1
+      subnet                = "private-b"
     },
     {
       server_name           = "celery0-staging"
       server_instance_type  = "t2.large"
-      subnet_index          = 1
+      subnet                = "private-b"
     },
     {
       server_name           = "pillow0-staging"
       server_instance_type  = "t2.large"
-      subnet_index          = 2
+      subnet                = "private-c"
     },
     {
       server_name           = "formplayer0-staging"
       server_instance_type  = "t2.micro"
-      subnet_index          = 0
+      subnet                = "private-a"
     },
     {
       server_name           = "kafka0-staging"
       server_instance_type  = "t2.medium"
-      subnet_index          = 0
+      subnet                = "private-a"
     },
     {
       server_name           = "es0-staging"
       server_instance_type  = "t2.micro"
-      subnet_index          = 0
+      subnet                = "private-a"
```

Comment:
Shouldn't elasticsearch have a separate subnet like we are doing in postgresql.
Or at least different than applications subnet.